File: 260911/task 6/code.py
import sys

a: float = float(input("A << "))
b: float = float(input("B << "))
c: float = float(input("C << "))

if a == 0:
    input("\nError! Not an quadratic function.\n\nPress ENTER to exit...")
    sys.exit()

D: float = b * b - 4 * a * c
print(f"\nDiscriminant >> {round(D,3)}")

if D > 0:
    print("Two roots!")
    
elif D < 0:
    print("Zero roots!")
    
else:
    print("One root!")
    
input("\nPress ENTER to exit...\n")

File: 260911/task 6/code.cs
namespace TestCS
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int a, b, c;
            float D;

            Console.Write("A << ");
            a = Convert.ToInt32(Console.ReadLine());
            Console.Write("B << ");
            b = Convert.ToInt32(Console.ReadLine());
            Console.Write("C << ");
            c = Convert.ToInt32(Console.ReadLine());

            if (a == 0)
            {
                Console.WriteLine("Error! Not an quadratic function.\n\nPress any key to exit...");
                Console.ReadLine();
                return;
            }

            D = b * b - 4.0f * a * c;

            Console.WriteLine($"\nDiscriminant >> {D:0.000}");

            if (D > 0)
            {
                Console.WriteLine("Two roots!");
            }
            else if (D < 0)
            {
                Console.WriteLine("Zero roots!");
            }
            else
            {
                Console.WriteLine("One root!");
            }

            Console.WriteLine("\nPress ENTER to exit...");
            Console.ReadLine();
        }
    }
}
